core/src/sigverify.rs

```diff
@@ -422,9 +414,9 @@ mod tests {
         let (sig_len, sig_start, msg_start_offset, pubkey_offset) =
             sigverify::get_packet_offsets(&packet, 0);
```

Comment:
The only test passes in zero.  My assumption is that if I passed in 10, we should add 10 to all outputs except `sig_len`?

Reply:
believe so